includes/tracks/events/class-wc-admin-setup-wizard-tracking.php

```diff
@@ -19,9 +19,54 @@ public static function init() {
 			return;
 		}
 
+		add_action( 'wc_setup_footer', array( __CLASS__, 'add_footer_scripts' ) );
+		add_action( 'admin_init', array( __CLASS__, 'track_marketing_signup' ), 1 );
 		self::add_step_save_events();
 	}
 
+	/**
+	 * Get the name of the current step.
+	 *
+	 * @return string
+	 */
+	public static function get_current_step() {
+		return isset( $_GET['step'] ) ? sanitize_key( $_GET['step'] ) : ''; // phpcs:ignore WordPress.Security.NonceVerification.NoNonceVerification
+	}
+
+	/**
+	 * Add footer scripts to OBW since it does not contain hooks for
+	 * wp_footer to allow the default methods of enqueuing scripts.
+	 */
+	public static function add_footer_scripts() {
+		echo '<script type="text/javascript" src="https://stats.wp.com/w.js?ver=' . gmdate( 'YW' ) . '"></script>'; // @codingStandardsIgnoreLine
+		wc_print_js();
+	}
+
+	/**
+	 * Track the marketing form on submit.
+	 */
+	public static function track_marketing_signup() {
+		if ( 'next_steps' !== self::get_current_step() ) {
+			return;
+		}
+
+		wc_enqueue_js(
+			"
+			var form = $( '.newsletter-form-email' ).closest( 'form' );
+			$( document ).on( 'submit', form, function() {
+				window.wcTracks.recordEvent( 'obw_marketing_signup' );
+			} );
+			$( '.wc-setup-content a' ).click( function trackNextScreen( e ) {
```

Comment:
This tracks all links in the content area (including the help text links).  Since the help text links open a new window, users who click those will probably have multiple next step events fired.

I think this is probably okay either way depending on how the data is being used, but do you think we should track all or just the onsite links?

```suggestion
			$( '.wc-wizard-next-steps a' ).click( function trackNextScreen( e ) {
```

Reply:
>users who click those will probably have multiple next step events fired

Right. I think thats ok. This way we'll know information about the help text links and any funnels created can regex for any just the next steps in the wizard.